### ISSUE TITLE
Farvardin 2,  is not recognized! 

### ISSUE THREAD

**Issue by babakrkb** [jalaali/moment-jalaali#203]
The object for the second day of Farvardin is always as the same object for the first day of Farvardin for all years. For example:

`moment("1398/01/02", "jYYYY/jMM/jDD").format("jYYYY/jMM/jDD") // retruns 1389/01/01`

What can I do for this propblem?

**Comment by behrang** [comment]
It must be a timezone problem. What is the location of your android phone?

**Comment by babakrkb** [comment]
Yes it is a timezone problem. When I change it to US timezone, it works fine, but when I change the timezone to Tehran, It does not work properly.

**Comment by behrang** [comment]
Add 12 hour to your input string, and parse that too. This is a way to have enough offset, so timezones cannot interfere.
